blocks/blockstore/blockstore.go

```diff
@@ -74,6 +74,10 @@ type blockstore struct {
 }
 
 func (bs *blockstore) Get(k key.Key) (blocks.Block, error) {
+	if k == "" {
+		return nil, ErrNotFound
```

Comment:
we could add an `ErrInvalidKey` but we'd then have to consider where it should be treated differently etc. may be more difficult than just this bugfix